dataframe/arrow_test.go

```diff
@@ -51,3 +52,32 @@ func TestEncode(t *testing.T) {
 		t.Fatalf("data frame doesn't match golden file")
 	}
 }
+
+func TestDecode(t *testing.T) {
+	df := dataframe.New("http_requests_total", dataframe.Labels{"service": "auth"},
```

Comment:
Since test encode uses this. It should be the same object. So maybe unify the tests, or make `df` outside the scope of the test.